sweepbatcher/log.go

```diff
@@ -17,7 +17,8 @@ func init() {
 	UseLogger(build.NewSubLogger("SWEEP", nil))
 }
 
-// batchPrefixLogger returns a logger that prefixes all log messages with the ID.
+// batchPrefixLogger returns a logger that prefixes all log messages with
```

Comment:
ultra nit: do you have the line break set to 80 chars? The `the` of `the ID` technically fits into the first line. Also seeing early line breaks elsewhere.

Reply:
The length of "// batchPrefixLogger returns a logger that prefixes all log messages with the ID." is 81 character. It has a period in the end.

I have the following settings in vim to highlight with red last few characters up until 80th. And if something does not fit into "red zone", it means the line is tool long.

```
match ErrorMsg /\%<81v.\%>77v/
```

Reply:
Ah ok, so this
```
// batchPrefixLogger returns a logger that prefixes all log messages with the 
// ID.
```
would be valid according to your settings?